### ISSUE TITLE
svg icon classes/id conflicts

### ISSUE THREAD

**Issue by indraraj26** [czeckd/angular-svg-icon#136]
if you have same id in other svg then the second one will not display. 

Is there anyway to force svg to have unique id (via encapsulation)

svg1: https://svgshare.com/i/_7P.svg
svg2: https://svgshare.com/i/_6v.svg

**Comment by czeckd** [comment]
No, see the open issue #73. Creating unique ids is a requested enhancement that I've not had time to implement as of yet.

**Comment by indraraj26** [comment]
I checked #73 but still it is hacky way and you will have to do it for all svg which is quite lot of works. 

Btw overall this is really great package. I appreciate your efforts to maintain this package.  

I was also thinking how we can give unique id to path, g, clipath is also there. We might need to find the way to encapsulate each svg-icon component just like angular does it with viewEncapsulation.emulated or default

If you think this can be close feel free to do it.

**Comment by Teebo** [comment]
@indraraj26 you can try this package: https://www.npmjs.com/package/ngx-svg-schematic

**Comment by kuncevic** [comment]
You can save the svg's rather with unique classes names or inline styles, that would fix that problem
